CONTRIBUTING.md

```diff
@@ -59,9 +55,28 @@ If your code requires new dependencies, be sure that those are available on all
 
 ### User experience
 
-Jamulus is used by people all over the world with different backgrounds and knowledge. Features and language should be user focused.
-The Jamulus project has a [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) which you should follow to remain consistent.
-Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+Jamulus is used by people all over the world with different backgrounds and knowledge. Therefore, think user focused.
+
+- Features should be usable in the sense that they act as expected to somebody who does not have a technical background.
+- Make sure you follow the [style and tone guide](https://jamulus.io/contribute/Style-and-Tone) to remain consistent.
+
+### Submitting code
+
+- Create a Github fork of the Jamulus code and create a Pull Request when you are done. Please don't use diff/patch to send your code changes. GitHub can be confusing, so feel free to ask for help.
+- Please run a local (build/feature) test to ensure there are no errors. After opening a pull request, keep an eye on the CI checks for quality or compile issues and fix them as required.
```

Comment:
Just an idea (not sure if a good one): We could also point people to using a `autobuild/foo` branch name and check their own repo's Actions before submitting the PR (instead of or in addition to building locally).
Note though that Workflows have to be enabled manually after forking (I think it's possible right via the Actions tab).

Reply:
I'd prefer not to write another "how to contribute to GitHub projects" file. This would make the problem even worse in my opinion (due to even more clutter around the web). Maybe just put a comment about the autobuild/ branching here.

Reply:
```suggestion
- Please run a local (build/feature) test to ensure there are no errors. After opening a pull request, keep an eye on the CI checks for quality or compile issues and fix them as required. You can also test the build on your repository by naming your branch `autobuild/<branchName>` to invoke the building process on your repo.
```